pinax/stripe/models.py

```diff
@@ -272,6 +276,27 @@ class BitcoinReceiver(StripeObject):
     used_for_payment = models.BooleanField(default=False)
 
 
+@python_2_unicode_compatible
+class Discount(models.Model):
+
+    coupon = models.ForeignKey("Coupon", on_delete=models.CASCADE)
+    customer = models.OneToOneField("Customer", null=True, on_delete=models.CASCADE)
+    subscription = models.OneToOneField("Subscription", null=True, on_delete=models.CASCADE)
+    start = models.DateTimeField(null=True)
+    end = models.DateTimeField(null=True)
+
+    def __repr__(self):
+        return "Discount(coupon={}, customer={}, subscription={})".format(self.coupon, self.customer, self.subscription)
```

Comment:
Should use the same style as with https://github.com/pinax/pinax-stripe/pull/382.
Especially it should use `{!r}`.

Also the customer is on the Subscription already:
1. it might not be necessary here then
2. if being kept here, it should come via the `repr` of the subscription only then.